src/libraries/System.Net.Quic/tests/FunctionalTests/QuicTestBase.cs

```diff
@@ -40,10 +42,31 @@ public abstract class QuicTestBase : IDisposable
         public const int PassingTestTimeoutMilliseconds = 4 * 60 * 1000;
         public static TimeSpan PassingTestTimeout => TimeSpan.FromMilliseconds(PassingTestTimeoutMilliseconds);
 
-        public QuicTestBase(ITestOutputHelper output)
+        private static bool s_queueDelaySet;
+        private static int s_initMessage;
+
+        static unsafe QuicTestBase()
+        {
+            if (IsSupported)
+            {
+                QUIC_SETTINGS settings = default(QUIC_SETTINGS);
+                settings.IsSet.MaxWorkerQueueDelayUs = 1;
+                settings.MaxWorkerQueueDelayUs = (uint)TimeSpan.FromSeconds(2.5).TotalMicroseconds;
```

Comment:
Seems weird to create TimeSpan just to convert seconds to microseconds. That is just point shift in decimal system right? And this could be `const`....

Reply:
Changed to number.